js-exercises/sequential-promise/sequentialPromise.js

```diff
@@ -0,0 +1,14 @@
+const sequentialPromise = fns => {
+    const resultantPromise = fns.reduce((promise, fn) => {
+      return promise.then(result => {
+        return fn(result[result.length - 1]).then(res => {
+          return result.concat(res);
+        });
```

Comment:
here you could use 'push' instead of 'concat'....just a suggestion

Reply:
@nihal1993 ......https://codeburst.io/jsnoob-push-vs-concat-basics-and-performance-comparison-7a4b55242fa9